cypress/commons/actions/generic/TableParameters.js

```diff
@@ -79,6 +83,10 @@ function getExportConfirmButton() {
   return getExportDialog().find(GENERIC_SELECTORS.genericComponents.table.export.confirmButton);
 }
 
+function getFullscreenButton(tableParameterElement) {
+  return getGrid(tableParameterElement).find(GENERIC_SELECTORS.genericComponents.table.toolbar.fullscreen);
```

Comment:
Could we rename the last key from `fullscreen` to `fullscreenButton`?

Reply:
DONE